maps/away/ships/freebooter/freebooter_salvager/freebooter_salvager_ghostroles.dm

```diff
@@ -0,0 +1,88 @@
+/datum/ghostspawner/human/freebooter_altered_crew
```

Comment:
this file is exactly the same as the other freebooter ghostroles file, just with type paths and tag changes

the salvager should just ghostroles from the other freebooter ship
this whole file should only have a comment like `// this ship uses ghostroles frommaps\away\ships\freebooter\freebooter_ship\freebooter_ship_ghostroles.dm`